components/api_request_helper/api_request_helper.cc

```diff
@@ -92,9 +119,25 @@ void APIRequestHelper::OnResponse(
       }
     }
   }
+
   url_loaders_.erase(iter);
-  std::move(callback).Run(response_code, response_body ? *response_body : "",
-                          headers);
+  if (!response_body) {
+    std::move(callback).Run(response_code, "", headers);
+    return;
+  }
+
+  auto raw_body = *response_body;
```

Comment:
why this copy? It seems unnecessary and counterproductive as mentioned in DM

Reply:
done